### DISCUSSION TITLE
Search results with author in sidebar

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#660] [Q&A] [answered]
On your blog there is an author box in the sidebar with the search page opened. How to I achieve that?

My search page is default and looks like that one in your example site, but there is no author box.

https://blog.natenom.com/search/?q=hugo%20theme

**Comment by razonyang** [comment, marked as answer]
There are no default widgets for search sidebar, you'll need to create the widget via hooks, such as 'layouts/partials/hooks/search/sidebar-end.html`:

```html
{{- partial "sidebar/profile" . }}
```

**Reply by Natenom** [reply]
Thank you :) Works.

Would have never found that. Thought, it was some default stuff.

**Reply by razonyang** [reply]
Yep, but there is nothing worth to add at this moment, the default widgets should be related to the search, but there is not~.
